### PR TITLE
Import/export monitors documentation

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?
Adds the import / export functionality documentation 

### Motivation
PM request.

### Preview link

* https://docs-staging.datadoghq.com/gus/export-monitor/monitors/manage_monitor/#exporting-and-importing-monitors

### CODE REVIEW

File: content/en/monitors/manage_monitor.md [hunk 1]
```diff
@@ -70,6 +70,27 @@ There are a few caveats regarding quoted fields:
...
 * You may NOT use wildcard search inside quoted strings: `"chef-client*"`, while valid syntactically, won't return a monitor titled `"chef-client failing"` because the `*` is treated literally.
 
+## Exporting and importing Monitors
+
+### Exporting
+
+Export your monitor’s configuration in JSON format in order to use it to manage and deploy monitors programmatically. You can export a monitor’s JSON configuration in one of two ways using the Datadog UI:
+
+1. Navigate to an existing monitor’s [Monitor Status page][5] and select **Export** from its [Settings menu][6] in the top right corner of the page.
+2. Click the Export Monitor button at the bottom of the screen when [creating a new monitor][7].
```
Comment (mark-azer) [inline content/en/monitors/manage_monitor.md]: Bold "Export Monitor"

File: content/en/monitors/manage_monitor.md [hunk 2]
```diff
@@ -70,6 +70,27 @@ There are a few caveats regarding quoted fields:
...
+
+### Importing
+
+Import a monitor JSON configuration from one Datadog accounts to another from within the Datadog UI by following these instructions:
```
Comment (mark-azer) [inline content/en/monitors/manage_monitor.md]: The word "accounts" is plural but should be singular

File: content/en/monitors/manage_monitor.md (hunk 1 above)
Comment (apigirl) [inline content/en/monitors/manage_monitor.md]:
```suggestion
2. Click the **Export Monitor** button at the bottom of the screen when [creating a new monitor][7].
```

File: content/en/monitors/manage_monitor.md (hunk 2 above)
Comment (apigirl) [inline content/en/monitors/manage_monitor.md]:
```suggestion
Import a monitor JSON configuration from one Datadog account to another from within the Datadog UI by following these instructions:
```

File: content/en/monitors/manage_monitor.md [hunk 3]
```diff
@@ -80,9 +101,9 @@ To edit an individual monitor, hover over it and use the buttons to the far righ
...
 
-## Manage Triggered Monitors with group-level granularity
+### Manage Triggered Monitors with group-level granularity
 
-You can mute or [resolve][2] triggered monitors in bulk using the [Triggered Monitors page][5]. It's similar to the [Manage Monitors page](#managing-monitors)-you can find monitors by their attributes using the same easy tickboxes or query syntax-but there are a few differences. Aside from only showing monitors with a triggered status (Alert, Warn, or No Data), the main difference is that the Triggered Monitors page shows a row for _each group_ (i.e. each reporting source) of each monitor.
+You can mute or [resolve][2] triggered monitors in bulk using the [Triggered Monitors page][9]. It's similar to the [Manage Monitors page](#managing-monitors)-you can find monitors by their attributes using the same easy tickboxes or query syntax-but there are a few differences. Aside from only showing monitors with a triggered status (Alert, Warn, or No Data), the main difference is that the Triggered Monitors page shows a row for _each group_ (i.e. each reporting source) of each monitor.
```
Comment (apigirl) [inline content/en/monitors/manage_monitor.md]:
```suggestion
You can mute or [resolve][2] triggered monitors in bulk using the [Triggered Monitors page][9]. It's similar to the [Manage Monitors page](#managing-monitors) - you can find monitors by their attributes using the same easy tickboxes or query syntax-but there are a few differences. Aside from only showing monitors with a triggered status (`Alert`, `Warn`, or `No Data`), the main difference is that the Triggered Monitors page shows a row for _each group_ (i.e. each reporting source) of each monitor.
```

**Pull request merged**